### PR TITLE
Feature/IO-571_refactor

### PR DESCRIPTION
**Pull request by egor-slatvinskiy-idf**

### CODE REVIEW

**Review by vladimir-hodaryonok-idf — APPROVED** [review]

**Review by elizaveta-seleznyova-idf — APPROVED** [review]

File: packages/presentation/lib/ui/movie_details/movie_details_widget.dart [hunk 1]
```diff
@@ -423,9 +420,8 @@ class _CastWidget extends StatelessWidget {
   final MovieDetailsScreenData screenData;
 
   const _CastWidget({
-    Key? key,
-    required this.screenData,
-  }) : super(key: key);
+    required this.screenData
```
Comment (yuliya-klescheva-idf) [inline packages/presentation/lib/ui/movie_details/movie_details_widget.dart]: why did you delete the key?

File: packages/presentation/lib/ui/movie_details/movie_details_widget.dart [hunk 2]
```diff
@@ -502,7 +498,7 @@ class _CastWidget extends StatelessWidget {
                           width: Dimens.size70,
                           child: Text(
                             e.characters,
-                            maxLines: 2,
+                            maxLines: _textMaxLines3,
```
Comment (yuliya-klescheva-idf) [inline packages/presentation/lib/ui/movie_details/movie_details_widget.dart]: you change 2 to 3, it's ok?

File: packages/presentation/lib/library/dimens/dimens.dart [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Dimens {
-  Dimens._();
+  const Dimens._();
 
   static const double size021 = .21;
```
Comment (yuliya-klescheva-idf) [inline packages/presentation/lib/library/dimens/dimens.dart]: 0.21 better, I think

File: packages/presentation/lib/library/images_utils/style_image.dart [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/Library/images_utils/images_utils.dart';
+
+FadeInImage imagePoster({
```
Comment (alexei-kruk-idf) [inline packages/presentation/lib/library/images_utils/style_image.dart]: maybe rename file, because strange name

**Review by alexei-kruk-idf — APPROVED** [review]

**Review by yuliya-klescheva-idf — APPROVED** [review]

File: packages/presentation/lib/ui/movie_details/movie_details_widget.dart [hunk 5]
```diff
@@ -18,6 +18,12 @@ import 'package:presentation/ui/movie_details/data/movie_details_screen_data.dar
 import 'package:presentation/ui/movie_details/details_arguments/movie_details_arguments.dart';
 import 'package:presentation/ui/movie_page/model/movie_row_data.dart';
 
+const _textMaxLines2 = 2;
+const _textMaxLines3 = 3;
+const _textMaxLines4 = 4;
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/ui/movie_details/movie_details_widget.dart]: move textMaxLines to dimens. it can be used anywhere else

**Review by sergei-mikhailovskii-idf — APPROVED** [review]
